src/include/connection.h

```diff
@@ -515,39 +515,46 @@ struct __wt_connection_impl {
      */
     WT_FILE_SYSTEM *file_system;
 
+/*
+ * Server subsystem flags.
+ */
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define WT_CONN_SERVER_CAPACITY 0x01u
```

Comment:
I think you need `WT_CONN_SERVER_ASYNC` here?

Reply:
At the end of the list line 556

Reply:
Yeah but if the idea behind this change is to move `WT_CONN_SERVER` flags to their own field then `WT_CONN_SERVER_ASYNC` should get moved too.